eva_submission/eload_ingestion.py

```diff
@@ -173,21 +155,28 @@ def check_variant_db(self, db_name=None, db_name_mapping=None):
                     conn=conn
                 )
 
-        self.eload_cfg.set(self.config_section, 'database', value=assembly_to_db_name)
-
-        with pymongo.MongoClient(self.mongo_uri) as db:
-            names = db.list_database_names()
-            for assembly_accession, db_info in assembly_to_db_name.items():
-                db_name = db_info['db_name']
-                if db_name in names:
-                    self.info(f'Found database named {db_name}.')
-                    self.info('If this is incorrect, please cancel and pass in the database name explicitly.')
-                    self.eload_cfg.set(self.config_section, 'database', assembly_accession, 'exists', value=True)
-                else:
-                    self.error(f'Database named {db_name} does not exist in variant warehouse, aborting.')
-                    self.error('Please create the database or pass in the appropriate database name explicitly.')
-                    self.eload_cfg.set(self.config_section, 'database', assembly_accession, 'exists', value=False)
-                    raise ValueError(f'No database named {db_name} found.')
+        # Create the databases if they do not exists. Then shard them.
+        collections_shard_key_map = {
```

Comment:
Suggest using a separate file for the sharding code for better organization.

Reply:
It's now in `eload_utils.py`